css/amp.css

```diff
@@ -151,7 +151,15 @@ html.i-amphtml-ios-embed {
   overflow: hidden !important;
 }
 
-i-amphtml-sizer {
+.i-amphtml-layout-awaiting-size {
+  overflow: visible !important;
+}
+
+.i-amphtml-layout-awaiting-size > * {
+  display: none !important;
```

Comment:
Ha! we can resize a `display: none` element? will we still measure it? and what its `layoutboxRect` would be? always at the top left of the viewport? 
Would `overflow: visible` still needed? I played with http://jsbin.com/hivelus/2/edit?html,css,output it seems to me, with `display: none` margin will not collapse even with `overflow: hidden`.